demisto_sdk/commands/common/hook_validations/lists.py

```diff
@@ -0,0 +1,52 @@
+from distutils.version import LooseVersion
+from typing import List
+
+from demisto_sdk.commands.common.errors import Errors
+from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
+    ContentEntityValidator
+
+FROM_VERSION_LISTS = '6.5.0'
+
+
+class ListsValidator(ContentEntityValidator):
+    def __init__(self, structure_validator, ignored_errors=False, print_as_warnings=False,
+                 json_file_path=None, **kwargs):
+        super().__init__(structure_validator, ignored_errors, print_as_warnings,
+                         json_file_path=json_file_path, **kwargs)
+        self.from_version = self.current_file.get('fromServerVersion')
+        self.to_version = self.current_file.get('toServerVersion')
```

Comment:
for consistency with the rest of the content items
```suggestion
        self.from_version = self.current_file.get('fromVersion')
        self.to_version = self.current_file.get('toVersion')
```

Reply:
Currently the lists example I was working with had the fromServerVersion field, see attached example
![image](https://user-images.githubusercontent.com/72340690/137643580-bd414446-e715-4d74-8bc8-d10f84331cbb.png)